### DISCUSSION TITLE
Feature Request: Splitting Editor Window

### DISCUSSION THREAD

**Discussion by kbvc** [dbeaver/dbeaver#15358] [Q&A] [answered]
There should be a feature to split the SQL Editor into two independent windows so you can run two independent queries side-by-side for easy comparison of results.  Or, I may want to show certain schemas in one of those windows in tabs as I am working on a query in the other query.

Thank you.

**Comment by p91paul** [comment, marked as answer]
You can open 2 sql editors, and put them side by side with drag&drop on their tab. Both vertical and horizontal split is supported, as well as dragging a tab to a new window if you drag it outside the main one

**Comment by kbvc** [comment]
My bad. My interface was not working correctly.  After resetting my workspace it is working now.  Thank you.
